### DISCUSSION TITLE
Documentation has no directory tree

### DISCUSSION THREAD

**Discussion by GuuoJuun** [toeverything/AFFiNE#2745] [Q&A]
### Description

创建的文档在左边菜单中没有体现，切换时只能只能点击全部页面然后选择文档切换，作为一个高频行为，这个使用效率和成本太高了

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by houjoe0829** [comment]
Currently, only the pages that have been added to favorites will display their internally linked pages on the left sidebar. This is mainly to avoid cluttering the sidebar.

**Reply by hxp0618** [reply]
I also really need all docs support for the file tree  ，In addition, I would like to ask, whether there is support for multi-note collaboration, such as I have a document A, I now need to write a document B, I need to open the document A in the left half of the layout, document B in the right half of the document B as a reference!
